### DISCUSSION TITLE
Job config JSON is invalid

### DISCUSSION THREAD

**Discussion by sam-globis** [speechmatics/community#7] [Community help & support] [answered]
Hello,

I am trying to run a transcript job using cURL command line (on OSX) and a config.json with the transcription and output configurations. However, I am receiving an error which I cannot figure out.

`{"code":400,"detail":"Job config JSON is invalid. Error: unable to validate schema: invalid character 'â' looking for beginning of value","error":"Job rejected"}
curl: (3) URL using bad/illegal format or missing URL
curl: (3) URL using bad/illegal format or missing URL
curl: (3) URL using bad/illegal format or missing URL
curl: (3) unmatched brace in URL position 1:
{
 ^`

The command line I used:

API_KEY=API_KEY
PATH_TO_FILE=FILE.mp4

curl -L -X POST "https://asr.api.speechmatics.com/v2/jobs/" \
    -H "Authorization: Bearer ${API_KEY}" \
    -F config=”$(cat ./config.json)” \
    -F data_file=@${PATH_TO_FILE} \

The JSON file contains these lines:

`
{
  "type": "transcription",
  "transcription_config": 
  {
    "language": "en",
    "output_locale": "en-US",
    "operating_point": "enhanced",
    "additional_vocab": [
    {"content": "G1", "sounds_like": ["G one"]}
    ]
  },
  "output_config": {
    "srt_overrides": {
      "max_line_length": 37,
      "max_lines": 2
    }
  }
}
`

Any advice would be greatly appreciated.

**Comment by TudorCRL** [comment, marked as answer]
Hi @sam-globis, thanks for trying out our API and getting in touch! It's great to get feedback and hear where people are having issues.

A couple of us here at Speechmatics tried out your code example to see if it would work for us. The main problem we encountered was that the quotes being used for `-F config=”$(cat ./config.json)” ` are not standard quotes and so cause some problems. When we changed those, it seemed to work. It might be worth trying that out, and also checking your config.json file for any non-standard characters. Sometimes if you copy things into certain text editors it converts the characters into non-standard formats. This can be a major gotcha sometimes (it's caught us out a few times!).

Let me know if that works for you, if not, we can try digging deeper into the problem.

**Reply by sam-globis** [reply]
Hi @TudorCRL I was so focused on the json file that I overlooked those quotes! Glad it was just that! 
Everything works as intended now. Thank you so much for pointing that out.

**Reply by TudorCRL** [reply]
No problem, glad I could be of help!
